### ISSUE TITLE
DragOverlay shadowed preview

### ISSUE THREAD

**Issue by bloodbird** [clauderic/dnd-kit#443]
Hello and thanks for all the work on the dnd-kit!
I've migrated from an old version to the latest @dnd-kit/core and @dnd-kit/sortable and it fixed some bugs I had with autoScrolling.

What I can't manage to bring back now, is the DragOverlay shadowed preview in a vertical sorted list when Dragging without releasing. It used to display the future location as a shadow, out of the box, but now it just doesn't.

Could a gentle soul point me to the right parameter I am missing or the behavior name of what I am looking for?
Sorry to pollute here, I didn't know where to get this info!

**Comment by Qadir134** [comment]
Have you found the solution? @bloodbird

**Comment by bloodbird** [comment]
> Have you found the solution? @bloodbird

So, when looking back on this project, I don't remember what I did that brought it back. I just see that I've used : 
`<DragOverlay dropAnimation={null}>`

Within dnd context, outside the Droppable but I don't think it helps at all sorry!

I also used on SortableItem :

```
const style = {
        transform: CSS.Transform.toString(transform),
        transition,
    };
```

That's the only thing I find but again, that may not have been the solution. I remember that I either randomly got it back without knowing how, or I switched to a different release until I got my fix + the shadow.
